tests/test_openpgp.py

```diff
@@ -902,18 +904,27 @@ def test_refresh_wkd(openpgp_env_with_refresh,
                     openpgp_env_with_refresh.import_key(f)
 
             if server_key_var is not None:
-                responses.add(
-                    responses.GET,
-                    'https://example.com/.well-known/openpgpkey/hu/'
-                    '5x66h616iaskmnadrm86ndo6xnxbxjxb?l=gemato',
-                    body=globals()[server_key_var],
-                    content_type='application/pgp-keys')
+                body_ = globals()[server_key_var]
+                content_type_ = 'application/pgp-keys'
```

Comment:
Regarding content type:

> The server **SHOULD use "application/octet-stream"** as the Content-Type for the data but clients SHOULD also accept any other Content-Type. The server MUST NOT return an ASCII armored version of the key.

Source: https://www.ietf.org/archive/id/draft-koch-openpgp-webkey-service-15.html#section-3.1

Reply:
I know that it was preexisting :-) But, good that it will change while you are at it 👍